test/orange/OrangeGeoTestBase.hh

```diff
@@ -98,10 +98,10 @@ class OrangeGeoTestBase : public Test
     SurfaceId find_surface(char const* label) const;
 
     // Surface name (or sentinel if no surface);
-    std::string id_to_label(SurfaceId) const;
+    std::string id_to_label(UniverseId uid, LocalSurfaceId surfid) const;
 
     // Cell name (or sentinel if no surface);
-    std::string id_to_label(VolumeId) const;
+    std::string id_to_label(UniverseId uid, LocalVolumeId volid) const;
```

Comment:
For the common case (single unit tracker) where `UniverseId` is zero, add an overload:
```c++
std::string id_to_label(LocalVolumeId volid) const { return this->id_to_label{UniverseId{0}, volid; }
```
and you can re-simplify a lot of the tests. Same with surface IDs.